libs/designsystem/src/lib/components/dropdown/dropdown.component.ts

```diff
@@ -144,25 +154,29 @@ export class DropdownComponent
 
   @HostBinding('class.is-opening')
   get _isOpening(): boolean {
-    return this.state === openState.opening;
+    return this.state === OpenState.opening;
   }
 
   @HostBinding('class.is-open')
   get isOpen(): boolean {
-    return this.state === openState.open;
+    return this.state === OpenState.open;
   }
 
   @HostBinding('class.align-end')
-  _alignEnd: boolean;
+  _openLeft: boolean;
 
   @HostBinding('class.align-top')
-  _alignTop: boolean;
+  _openUp: boolean;
 
-  private set _horizontal(value: 'start' | 'end') {
-    this._alignEnd = value === 'end';
+  private set _horizontal(alignment: HorizontalDirection) {
+    this._openLeft = alignment === HorizontalDirection.left;
+  }
+  private get _horizontal() {
+    return this._openLeft ? HorizontalDirection.left : HorizontalDirection.right;
   }
-  private set _vertical(value: 'up' | 'down') {
-    this._alignTop = value === 'up';
+
+  private set _vertical(direction: VerticalDirection) {
+    this._openUp = direction === VerticalDirection.up;
```

Comment:
Consider flipping this and using `_horizontal` and `_vertical` as single-source-of-truth.
The HostBinding property can then be used as a boolean as to whether the css class should be applied or not:
```ts
private _horizontal: HorizontalDirection;

@HostBinding('class.align-end')
  get _openLeft() {
    return _horizontal === HorizontalDirection.left;
  }
```

Reply:
👍  simplified